paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -572,6 +605,20 @@ internal class PaymentSheetViewModel @Inject internal constructor(
         }
     }
 
+    private fun PaymentSelection.New.canSave(): Boolean {
```

Comment:
Should we extract this logic into a helper, make it internal, and test in isolation?